### PR TITLE
shimose test desu

### PR DESCRIPTION
**Pull request by mshimose**

### CODE REVIEW

**Comment by devinfra** [comment]
Can one of the admins verify this patch?

**Comment by devinfra** [comment]
Can one of the admins verify this patch?

File: shi.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+cent6test1
```
Comment (ttanimichi) [inline shi.txt]: hoge

Comment (ttanimichi) [inline shi.txt]: fuga

Comment (ttanimichi) [inline shi.txt]: hoge

**Pull request merged**